mantidproject/mantid
# Fix regression in GSASIIRefineFitPeaks and enable tests

Previously, the unit tests for GSASIIRefineFitPeaks did not run on the build servers, as they did not have GSAS-II installed. This PR fixes a bug which, for this reason, went unnoticed.

The unit test was converted to a system test to be run on the nightly server. Scripts for installing GSAS-II automatically were implemented to make this as painless as possible.

**To test:**

Install GSAS-II using one of the new scripts, then make sure the system test is not skipped and passes. Also code review.

Fixes #21482

**Release Notes** 

Doesn't need to be in the release notes, as it's an internal change, and no scientists were aware that the algorithm was unstable in the first place

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
